### PR TITLE
[DOCS] Clarify docs: how to add Routing enforcement

### PR DESCRIPTION
**Pull request by cjcjameson**
In 6.8, the `_doc` made sense: https://www.elastic.co/guide/en/elasticsearch/reference/6.8/mapping-routing-field.html

but now it doesn't, and is confusing. I assume it applies to all documents(??)

### CODE REVIEW

**Comment by cjcjameson** [comment]
I did sign before submitting; feel free to refresh the CLA verification.

**Comment by jrodewig** [comment]
HI @cjcjameson 

Can you ensure that you have signed the CLA using the same **@users.noreply.github.com** email address associated with this commit?

That's likely causing the CLA check to fail. Thanks!

**Comment by jrodewig** [comment]
@elasticmachine test this please

**Review by jrodewig — APPROVED** [review]
LGTM.

I'll merge after the CLA CI test passes.

**Comment by cjcjameson** [comment]
@jrodewig thanks! I changed the author and also repointed this MR to the `master` branch. Can you do the backporting? What's the system for backporting docs changes to the right branches?

Also filed #53991 regarding the CLA

**Comment by jrodewig** [comment]
@cjcjameson Thanks. It looks like that solved the CLA issue.

I'm going to change the base branch back to 7.6. That'll let me merge it and backport the changes.
For this repo, the repo maintainers (me in the case) handle the redirects. I'll get this applied to doc branches we still maintain.

Thanks again!

**Comment by jrodewig** [comment]
@elasticmachine test this please.

**Comment by jrodewig** [comment]
Switched base branch back to master. That seemed to clear up the merge conflicts. Will merge after CI is complete.

**Pull request merged**
